### ISSUE TITLE
Can the server Push the location ?

### ISSUE THREAD

**Issue by marifrahman** [miataru/miataru-server#2]
Just wondering if the server be a Push Notification server if the locations.

**Comment by bietiekay** [comment]
currently it cannot push information to the client. But with the current infrastructure behind miataru it should not be a big issue to add this functionality. 

Would you be interested to help adding it?

**Comment by marifrahman** [comment]
yep -would love to contribute. I am going through the project - please let
me know if you have any suggestions for it.
